internal/uidriver/glfw/ui.go

```diff
@@ -821,7 +822,15 @@ event:
 			time.Sleep(time.Millisecond)
 		}
 	}
-	window.SetFramebufferSizeCallback(glfw.ToFramebufferSizeCallback(nil))
+	if u.defaultFramebufferSizeCallback == 0 {
+		// When window gets resized (either by manual window resize or a window
+		// manager), glfw sends a framebuffer size callback which we need to handle
```

Comment:
* Add the issue number.
* This problem is confirmed by i3 window manager but not by other managers, right?

So the comment should be like this:

```
		// When the window gets resized (either by manual window resize or a window
		// manager), glfw sends a framebuffer size callback which we need to handle (#1960).
		// This event is the only way to handle the size change at least on i3 window manager.
```

Reply:
> This problem is confirmed by i3 window manager but not by other managers, right?

Yes, but I might try other tiling WMs, I'm just not used to other WMs. 
Also tested that xfwm (default XFCE window manager, not a tiling WM) works fine with this patch too